customerio-reactnative.podspec

```diff
@@ -28,7 +28,9 @@ Pod::Spec.new do |s|
   s.default_subspec = "nopush"
 
   s.subspec 'nopush' do |ss|
-    # no dependencies. This is the default subspec designed to not install any push dependencies.
+    # This is the default subspec designed to not install any push dependencies. Customer should choose APN or FCM.
+    # The SDK at runtime currently requires the MessagingPush module so we do include it here. 
+    ss.dependency "CustomerIO/MessagingPush", package["cioNativeiOSSdkVersion"]
```

Comment:
Looking at the comment, I'm not clear on what the customer needs to do.  What about adding both APN and FCM options as comments and directing the customer to uncomment the one they need and deleting the other?

Reply:
> I'm not clear on what the customer needs to do.

The customer needs to follow our SDK docs to setup APN or FCM push. 

This PR is trying to handle the scenario where a customer has not yet setup APN or FCM push. Some customers have sent us support tickets saying that they are experiencing issues with our SDK while following our SDK docs. They must be doing 1 step at a time, continuously compiling their app and testing along the way. If a customer does that, they could encounter issues. This PR is to try and fix those issues. 

This PR does not effect our docs today. No work on customers part besides what they already do.